<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewPLCCtrl</class>
 <widget class="QDialog" name="NewPLCCtrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>303</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>137</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>239</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_sure">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_sure_2">
      <property name="text">
       <string>帮助</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="DefaultSet">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>260</y>
     <width>231</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>恢复默认设置</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>561</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>连接属性</string>
   </property>
   <widget class="QLineEdit" name="edit_name">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>链接名称中不能包含'#'号</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>99</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_number">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>连接编号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>连接名称：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_number">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_HMInumber">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_HMInumber">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>本机编号：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_serialport">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>网络</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CAN1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CAN2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_serial">
    <property name="geometry">
     <rect>
      <x>311</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>连接端口：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>251</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>协议属性</string>
   </property>
   <widget class="QComboBox" name="stopbits">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="combo_baudrate">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>230400</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_check">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>检  验：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_checkType">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>NONE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EVEN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ODD</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_kp">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Kbps</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_databits">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_bit">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数据位：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_stopbit">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>停止位：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_baudrate">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>穿透设置</string>
   </property>
   <widget class="QCheckBox" name="check_transmit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>85</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>是否穿透</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="m_MainScreenradioBtn">
       <property name="text">
        <string>是否是主屏</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_FromScreenradioBtn">
       <property name="text">
        <string>是否是从屏</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="m_LinkFromScreenradioBtn">
       <property name="text">
        <string>连接从屏口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_LinkPLCradioBtn">
       <property name="text">
        <string>连接PLC</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>从站数量：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="m_FromScreenspinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
